source/mir/combinatorics/package.d

```diff
@@ -251,6 +300,9 @@ Params:
 
 Returns:
     Forward range, which yields the permutations
+
+See_Also:
+    $(LREF Permutations), $(LREF dispose)
```

Comment:
As it is bundled with `makePermutations`, linking to `dispose` could be useful to the user